src/flapjack/transport_test.go

```diff
@@ -12,4 +12,28 @@ func TestDialFails(t *testing.T) {
 	}
 }
 
-// TODO(auxesis): add test for sending and receiving Events
+func TestSendSucceeds(t *testing.T) {
+  transport, _ := Dial("localhost:6379", 9)
```

Comment:
Do we want to mock redis rather than relying on a redis to be available at localhost:6379 for these tests to pass? /cc @ali-graham @auxesis